src/CMakeLists.txt

```diff
@@ -155,7 +155,7 @@ include(CTest)
 include(Catch)
 target_link_libraries(tests ${sfml_libs} ${rclcpp_LIBRARIES} ${geometry_msgs_LIBRARIES})
 target_link_libraries(tests Catch2::Catch2)
-target_link_libraries(tests ${OpenCV_LIBS})
+target_link_libraries(tests ${OpenCV_LIBS} opencv_aruco)
```

Comment:
Interesting, I would have thought `opencv_aruco` would be included in `${OpenCV_LIBS}`. Did the linking error only happen on the Jetson, or on other machines as well?

Reply:
Oddly, it happens on my laptop. This may be an artifact of some weird way I installed OpenCV in eons past. I'm not sure about the Jetson though---I haven't tried building `tests` there.